pykern/pkcli/ci.py

```diff
@@ -8,25 +8,56 @@
 :copyright: Copyright (c) 2022 RadiaSoft LLC.  All Rights Reserved.
 :license: http://www.apache.org/licenses/LICENSE-2.0.html
 """
-from pykern.pkdebug import pkdp, pkdlog
```

Comment:
This line states in, because it makes debugging convenient.

Reply:
e987d4cecc20f01b5a8d29ed4f44274e61aaeb0a